roles/dropwizard-api/meta/main.yml

```diff
@@ -3,3 +3,5 @@
     - { role: openjdk-7, become: yes }
     - { role: git, become: yes }
     - { role: supervisord, become: yes }
+    - { role: mysql-client, become: yes }
+    - { role: mysql-server, become: yes }
```

Comment:
Let's separate the mysql-server into another ansible role. Not all the dropwizard-api servers need to have mysql-server installed. Only in the dev dropwizard-api servers we'll need to install mysql-server in production, we won't.